cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,73 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  let user;
 
+  beforeEach(() =>{
+    cy.visit('/');
+    cy.task('generateUser').then(generateUser => {
+      user = generateUser;
+    });
   });
-});
+
+  it('Assert the pagination and rows count selection', () => {
+    cy.get('.pagination-bottom').should('exist');
+    cy.get('.-next').click();
+    cy.get('.-previous').click();
```

Comment:
```suggestion
    cy.get('.-next').click();
    cy.get('.-previous').click();
```
what do you check here? you just write actions "click" but assert nothing. You need to assert that page 2 is opened when you clicks button 'next' and then assert that page 1 is opened after the clicking button "previous"